programs/mpl-inscription/src/state.rs

```diff
@@ -43,7 +43,8 @@ pub struct InscriptionMetadata {
     pub inscription_bump: Option<u8>,
     pub update_authorities: Vec<Pubkey>,
     pub associated_inscriptions: Vec<AssociatedInscription>,
-    pub _padding: [u8; 8],
+    pub mint: Option<Pubkey>,
+    pub _padding: [u8; 7],
```

Comment:
Because the padding is explicitly set, if I were to just add the mint here it would fail deserialization because the deserializer would look for a padding array of length 8, but only find one of length 7. The first byte is being repurposed for the mint Option. I preferred this method of explicit padding to just doing a +8 in the alloc step because this felt clearer than doing it that way. But I'm open to feedback on doing this in the future.

Reply:
Yeah I understand now thanks!